README.md

```diff
@@ -419,6 +419,7 @@ Awesome Wagtail [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38
 
 - [bakerydemo](https://github.com/wagtail/bakerydemo) – Next generation Wagtail demo, born in Reykjavík.
 - [wagtaildemo](https://github.com/wagtail/wagtaildemo) – An example site implemented with Wagtail.
+- [guide](https://github.com/wagtail/guide) – A website to teach Wagtail CMS to content editors, moderators and administrators.
```

Comment:
Please can new items be put to the end of the list in this section. 

https://github.com/springload/awesome-wagtail/blob/master/.github/CONTRIBUTING.md

> Additions should be added to the bottom of the relevant category.

Reply:
In guide, should we keep Wagtail or Wagtail CMS. As per contributing guidelines, It should be Wagtail only but I have seen above Open source sites, few of them included Wagtail CMS.

Reply:
Good call out. Maybe worth cleanup in a separate PR